README.md

```diff
@@ -1,96 +1,132 @@
-
-# ensime-vim
+# Ensime for vim `ensime-vim`
 
 [![Join the chat at https://gitter.im/ensime/ensime-vim](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ensime/ensime-vim?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Build Status](https://drone.io/github.com/yazgoo/ensime-vim/status.png)](https://drone.io/github.com/yazgoo/ensime-vim/latest)
 [![Coverage Status](https://coveralls.io/repos/yazgoo/ensime-vim/badge.svg?branch=master&service=github)](https://coveralls.io/github/yazgoo/ensime-vim?branch=master)
 
-ENSIME for the Editor of the Beast (Vim)
-
-# demo
+Vim, the editor of the Beast, has got an **Ensime** plugin to convert your preferred text editor
+into a Java/Scala **IDE**. It's not time for half measures and `ensime-vim` aims at
+becoming your go-to option to easily edit your source code in an efficient and fast way.
 
-![alt tag](https://raw.github.com/yazgoo/ensime-vim/master/doc/demo.gif)
+# Demo
 
-# howto
+![First demo](doc/ensime-vim.gif)
+![Second demo](doc/ensime-vim2.gif)
```

Comment:
Ah, and BTW, most of vim plugins uses two images to show the functionality. I wanted to keep that "standard" style.